linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/EmitterNames.scala

```diff
@@ -19,14 +19,11 @@ private[emitter] object EmitterNames {
   val UndefinedBehaviorErrorClass =
     ClassName("org.scalajs.linker.runtime.UndefinedBehaviorError")
 
-  /* In theory, some of the following could be computed from the Class
+  /* In theory, java.lang.CharSequence could be computed from the Class
    * Hierarchy. However, that would be require dealing with incremental runs,
-   * which would be overkill since these things are in fact known to be static.
+   * which would be overkill since it is in fact known to be static.
    */
```

Comment:
Doesn't this comment also apply to the method names below?

Reply:
Ah, that's a good point. I'll revert the change.